R/shapley.R

```diff
@@ -301,6 +301,7 @@ gaussian_transform <- function(x) {
 #' @return data.table with \code{noSamp_MC} (conditional) Gaussian samples
 #'
 #' @import condMVNorm
+#' @import mvnfast
 #' @export
```

Comment:
This is not necessary. Remove the following lines (303 & 304):
```
#' @import condMVNorm
#' @import mvnfast
```
Note that you should regenerate documentation after doing this.

Reply:
Done. I thought this was the proper way to add packages that should be imported?

Reply:
It is not necessary when we're using `mvnfast::foo()`. And if it is only one function you'd like to import (and you're using it in multiple cases) it is actually better to use `@importFrom mvnfast foo`. Then you could change `mvnfast::foo()` to `foo()`. 

But I would recommend to use `mvnfast::foo()` due to two reasons: 
1) You'll immediately know that `foo()` is a function from `mvnfast`   
2) You avoid potential problems if there are cases where two (or more) packages have a function that shares the same name